src/test/java/seedu/address/storage/JsonAdaptedStudentTest.java

```diff
@@ -36,6 +36,14 @@ public void toModelType_validStudentDetails_returnsStudent() throws Exception {
         assertEquals(BENSON, student.toModelType());
     }
 
+    @Test
+    public void toModelType_invalidUuid_throwsIllegalValueException() {
```

Comment:
Apologies, mistake on my part for not typing correctly. Should be
```suggestion
    public void toModelType_nullUuid_throwsIllegalValueException() {
```